tests/unittests/tests-credman/credentials.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       credman test application (PSK and ECC keys)
```

Comment:
This file does not contain an application, just the `credential` (just move the line below here).